en/docs/guides/login/using-jwks.md

```diff
@@ -1,8 +1,8 @@
-# Call the JSON Web Key Set (JWKS) Endpoint
+# Obtain Key Set Using JSON Web Key Set
 
 The [JSON Web Key Set (JWKS) endpoint](http://openid.net/specs/openid-connect-discovery-1_0.html) is a read-only endpoint that returns the Identity Server's public key set in the [JWKS](../../../concepts/authentication/jwks) format. 
 
-This contains the signing key(s) that the Relying Party (RP) uses to validate signatures from WSO2 Identity Server.
+You can refer this JWKS documentation, When your Relying Party (RP) application needs to validate the signatures of the JWT Tokens issued by the WSO2 Identity Server.
```

Comment:
You can follow this guide when your relying party (RP) application needs to validate JWT Token signatures issued by WSO2 Identity Server.